client/src/rr/dnssec/keypair.rs

```diff
@@ -241,11 +242,12 @@ impl KeyPair {
     /// # Return
     ///
     /// the KEY record data
+    #[allow(deprecated)]
     pub fn to_sig0key(&self, algorithm: Algorithm) -> DnsSecResult<KEY> {
         self.to_public_bytes()
             .map(|bytes| {
                      KEY::new(Default::default(),
-                              Default::default(),
+                              KeyUsage::Zone,
```

Comment:
I'm in favor of the dual function approach, where anyone can has in `KeyUsage` if needed. I'd prefer to keep the standard option default and add some comments about compatibility... My concern isn't as much matching BIND as it is being compliant with the RFCs. I think the docs can explain this.

Would you mind adding that other function, and then making the existing one just call through to that with a default `KeyUsage`?